### PR TITLE
image cropping

### PR DESCRIPTION
**Pull request by Omid07**

### CODE REVIEW

File: app/src/main/java/com/example/omid/ps01/EditImageActivity.java [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
-            mImage.setImageBitmap(BitmapFactory.decodeFile(imagePath));
-        }
+        mFragmentManager = getFragmentManager();
+        mMainFragment = new MainFragment();
+        addFragment(mMainFragment);
+        mCrop= (Button) findViewById(R.id.button_crop);
+        mCrop.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                CropFragment cropFragment = new CropFragment();
```
Comment (ahammad-reyad) [inline app/src/main/java/com/example/omid/ps01/EditImageActivity.java]: refactor code and use replaceFragment() method

File: app/src/main/java/com/example/omid/ps01/EditImageActivity.java [hunk 2]
```diff
@@ -1,27 +1,65 @@
...
-}
+
+    private void addFragment(Fragment fragment) {
+        android.app.FragmentTransaction fragmentTransaction = mFragmentManager.beginTransaction();
+        fragmentTransaction.add(R.id.frame_layout, mMainFragment);
+        fragmentTransaction.commit();
+    }
+
+    public void replaceFragment() {
+        Fragment fragment = new MainFragment();
```
Comment (ahammad-reyad) [inline app/src/main/java/com/example/omid/ps01/EditImageActivity.java]: refactor code to take fragment as parameter

File: app/src/main/res/layout/activity_edit_image.xml [hunk 3]
```diff
@@ -6,20 +6,27 @@
...
         android:paddingTop="@dimen/activity_vertical_margin"
         tools:context=".EditImageActivity">
 
-        <ImageView
-            android:id="@+id/image_edit"
+        <FrameLayout
+            android:id="@+id/frame_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            ></FrameLayout>
```
Comment (nvmanh) [inline app/src/main/res/layout/activity_edit_image.xml]: format code?

**Comment by nvmanh** [comment]
OK!

**Pull request merged**